cmd/fs-v1-multipart.go

```diff
@@ -75,12 +75,12 @@ func (fs fsObjects) removeUploadID(bucket, object, uploadID string, rwlk *lock.L
 	// Check this is the last entry.
 	if uploadIDs.IsEmpty() {
 		// No more uploads left, so we delete `uploads.json` file.
-		return fs.deleteUploadsJSON(bucket, object, uploadID)
+		return true, fs.deleteUploadsJSON(bucket, object, uploadID)
```

Comment:
I see there is no need to return `(bool, error)` because  `bool` is exactly the same result of `err == nil`

Reply:
@balamurugana 

don't u think we need a explicit flag to express uploadIDs is empty?

Reply:
even `err == nil`， we have two kinds of return:
1. false, nil -- has any uploads left
2. true, nil -- has no uploads left

logically, we should clean up the Dirs with result 2

Reply:
I was thinking about why not `removeUploadID()` (or a cleanup function) is not doing the object directory cleanup than handling at individual methods.  As we pass `rwlk` to `removeUploadID()` (which is not clean), this prevents doing that.

As this kind of fix is not the scope of this PR, we could take this PR in.